R/Simulations-methods.R

```diff
@@ -977,6 +977,17 @@ setMethod("show",
         dfNames = dfNames
       )
 
+      # report stopping rules
+      # Report individual stopping rules with non-<NA> labels.
+      stop_pct <- colMeans(object@stop_report) * 100
```

Comment:
Can you please add tests that check whether this reporting works as expected? including potential edge cases. Since this is the main feature of this PR